### ISSUE TITLE
 submitters should not be able to edit once the submission is ready for review.

### ISSUE THREAD

**Issue by carolyncole** [pulibrary/pdc_describe#1773]
Once a work has been submitted the user should not be able to edit the work.  This validation should occur at the controller level and not allow the user to edit (when they hit back for example).

Completion criteria:

- [ ] As a submitter when I got to edit a work that is awaiting approval I should get redirected to work show page and see a message "Your work is waiting for curator approval and may not be updated.  Contact the curators to allow you to edit the work"
- [x] Note: There is no button for the curator at this point.  That is NOT part of this ticket.  We will wait and see if it is really needed.

**Comment by carolyncole** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/RDSS-61a4f1a12a399b001730f65a/issues/pulibrary/pdc_describe/1773?planning-poker) with Zenhub @hectorcorrea @JaymeeH @jrgriffiniii @leefaisonr

**Comment by carolyncole** [comment]
I think this is the reason for the edit...
<img width="771" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMmR0QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--32c4eee577325c3d947048ee28faded3264140f1/Screenshot%202024-04-17%20at%201.11.01%E2%80%AFPM.png" alt="Screenshot 2024-04-17 at 1.11.01 PM.png" />

**Comment by carolyncole** [comment]
The edit also exists.  I think we keep a user from editing an approved work, but not one awaiting approval...
